bin/pem-python-default

```diff
@@ -6,3 +6,7 @@ pem-default() {
     echo "Make '$1' Python's version as default."
     pyenv global "$1" && pyenv shell "$1"
 }
+
+usage() {
+    echo "Usage: pem python-default [<version>]"
```

Comment:
@AntonKarabaza 

1. Please use standard formatting for variables like in https://github.com/extsoft/pem/blob/master/bin/pem
2. Describe what is `version`. Take a look for master commit to get some examples.